library/psa_crypto.c

```diff
@@ -3921,7 +3921,7 @@ static psa_status_t psa_cipher_setup( psa_cipher_operation_t *operation,
     if( ret != 0 )
         goto exit;
 
-#if defined(MBEDTLS_DES_C)
+#if defined(PSA_WANT_KEY_TYPE_DES)
```

Comment:
Is this dependent on the sw implementation? It seems this would also be needed for a hardware implementation, or perhaps I'm misunderstanding how the HW implementations will hook in.